CONTRIBUTING.md

```diff
@@ -100,6 +101,50 @@ Be sure to include any related GitHub issue references in the commit message.  S
 [GFM syntax](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) for referencing issues
 and commits.
 
+
+[dronevic]:https://ci.vmware.run/vmware/vic
+[e2edronevic]:https://e2e.ci.vmware.run/vmware/vic
+[dronesrc]:https://github.com/drone/drone
+[dronecli]:http://readme.drone.io/devs/cli/
+
+## Automated Testing
+
+Automated testing uses [Drone][dronesrc].
```

Comment:
Robot definitely does not require drone, but drone does setup some environment variables that the scripts rely on now like ESX_URL, ESX_USERNAME and ESX_PASSWORD. I have to think some on the nicest way to implement that for the end user.

Running locally isn't quite ready for prime time just yet because I was focused on getting us CI for our builds that can provide benefit instead of failing constantly like we currently have.  But potentially locally, with drone it could just be:
1. Create a secrets.yml file with the env variables you need.
2. Run drone exec -E secrets.yml -yaml integration.yaml


Mikkel Hagen
Quality Engineering Lead
Phone: (716) 307 0362
Address: 6500 River Pl Blvd, Austin, TX 78730

On 05/28/2016 07:33 PM, Faiyaz Ahmed wrote:

In CONTRIBUTING.md<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_vic_pull_707-23discussion-5Fr64996099&d=CwMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=RWOEbpwLdzFOxVxJx1-lNmFIV0bt5SZaCb_OGY7aD4c&m=Cubno5p2tKACcA6ksO-4RiLr-l3rdArWoS-XtbdKGok&s=u6ufSM_6eSBfPeImF5AaIoV3j-AsCRc5CVl_3ew7YjQ&e=>:

> @@ -100,6 +101,50 @@ Be sure to include any related GitHub issue references in the commit message.  S
>  [GFM syntax](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) for referencing issues
>  and commits.
>
> +
> +[dronevic]:https://ci.vmware.run/vmware/vic
> +[e2edronevic]:https://e2e.ci.vmware.run/vmware/vic
> +[dronesrc]:https://github.com/drone/drone
> +[dronecli]:http://readme.drone.io/devs/cli/
> +
> +## Automated Testing
> +
> +Automated testing uses [Drone][dronesrc].


Does ROBOT they require drone? Can i run this through docker locally without drone? I ask because we have a build step which builds all of our binaries via docker (which is very convenient). Can we add the ROBOT stuff to the makefile so it's a buildstep which doesn't require drone?

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_vic_pull_707_files_d3c7fb8e6e6916ae2d1accbcaad7e9384f649be5-23r64996099&d=CwMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=RWOEbpwLdzFOxVxJx1-lNmFIV0bt5SZaCb_OGY7aD4c&m=Cubno5p2tKACcA6ksO-4RiLr-l3rdArWoS-XtbdKGok&s=_jRgVLjDh0rtU8il-RiszNuTRCXA_FTaCpjMDWTcjMQ&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe_ASfq-2DVi3HWEEDj209man8Uk1SjMokizUks5qGN7sgaJpZM4Igvq6&d=CwMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=RWOEbpwLdzFOxVxJx1-lNmFIV0bt5SZaCb_OGY7aD4c&m=Cubno5p2tKACcA6ksO-4RiLr-l3rdArWoS-XtbdKGok&s=nMpLz8ED3CmCyDGMYsTS0_rlkT2G-cv1gIGRrqDl-bQ&e=>.